pset-functions.js

```diff
@@ -208,6 +268,41 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+
+
+
+function doubleLetters(word) {
+ // split the word into an array of Letters
+
+ const letters = word.split('')
+ console.log(letters)
+
+ // loop through array of doubleLetters
+ //check if each letter is the same as the next letter
+ // in the sequence
+
+  let results = false
```

Comment:
@Ashbk32 algorithm is solid, however, this variable looks like it should be `result` and not `results`